spec/rubocop/cop/rspec/identical_equality_assertion_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IdenticalEqualityAssertion do
+  let(:msg) do
+    'Identical expressions on both sides of the equality '\
+    'may indicate a flawed test.'
+  end
```

Comment:
```suggestion
  let(:msg)  { described_class::MSG }
```

But I actually prefer having the message spelled out in each example. Yes, it's duplication, but a spec without method calls and interpolation is faster/easier to understand.

Reply:
I think we have an internal cop that prevents this:

```
spec/rubocop/cop/rspec/identical_equality_assertion_spec.rb:4:15: C: InternalAffairs/UselessMessageAssertion: Do not specify cop behavior using described_class::MSG.
  let(:msg) { described_class::MSG }
              ^^^^^^^^^^^^^^^^^^^^
```

Reply:
>But I actually prefer having the message spelled out in each example.

I'm not sure if you saw that part of my comment 😅